### DISCUSSION TITLE
In addition to ToonSquid’s “default” number of backups Projects should have own backup number

### DISCUSSION THREAD

**Discussion by rowby** [keiwando/toonsquid#436] [Ideas]
Hello Everybody,
I am working on very simple project that has no heavy duty “objects”.  That is, no jpgs, pngs, no MP4’s etc.
For that project I would like to have 20 Automatic Backups.  And I would want to see in that Project’s properties how much storage space those 20 Automatic Backups are using up in my iPad — because I might want to decrease or even increase the number of Automatic Backups for that specific project.  Right now, I am “flying blind”.

Most of us have limited storage on our iPads, so we would not want to  not overdo our Automatic Backups for some big heavy projects — but also want to have more backup protection for our small projects.

In addition to the above, I would like to see a summary of all projects’s automatic backups — which would allow us to delete or do an external backup of perhaps “old projects” - and let us free up our IPad’s precious storage of those no longer needed Automatic Backups.

I realize doing this would take a lot of programming time for our Beloved Keiwan..  However, if there is an “easy way” to do that I think most of us would like what I am suggesting.

Rowby

**Comment by keiwando** [comment]
Thanks for the suggestion, Rowby! I've noted it down and will consider it for future updates.
For now, you can go to `On My iPad > ToonSquid > Backup` in the `Files` app and find all of your backup folders there. You can long press them and tap on "Get Info" to see how much storage space the backups for each project are taking up.
